nacc_form_validator/errors.py

```diff
@@ -88,7 +89,8 @@ def __set_custom_error_codes(self):
             0x1009:
             "{1} for if {2} else {3} - compatibility rule no: {0}",
             0x2000:
-            "{1} in current visit for {2} in previous visit - temporal rule no: {0}",
+            "{1} in current visit for if {2} in previous visit then {3} " +
+            "in current visit - temporal rule no: {0}",
```

Comment:
Remove "in current visit" =>  "{1} for if {2} in previous visit then {3} in current visit - temporal rule no: {0}".
Do the same in 3004